cmd/cosign/cli/verify/verify.go

```diff
@@ -142,7 +142,7 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 		co.TSACerts = tsaCertPool
 	}
 
-	if keylessVerification(c.KeyRef, c.Sk) {
+	if !c.SkipTlogVerify {
```

Comment:
QQ: `!c.SkipTlogVerify` should gate the Rekor stuff, but `keyless` should still gate the fulcio CA's, right? See my PR here (aaah conflicts!) https://github.com/sigstore/cosign/pull/2393

Reply:
Oh yea, good catch. Adding back keylessVerification for Fulcio.